packages/core/src/Stores/Helpers/client.js

```diff
@@ -6,6 +6,7 @@ export const getClientAccountType = loginid => {
     if (/^VR/.test(loginid)) account_type = 'virtual';
     else if (/^MF/.test(loginid)) account_type = 'financial';
     else if (/^MLT|MX/.test(loginid)) account_type = 'gaming';
+    else if (/^VRTC/.test(loginid)) account_type = 'demo';
```

Comment:
there is no difference between `demo` and `virtual` , so no need for this

Reply:
Thanks, Thought there were no condition for it.